### DISCUSSION TITLE
Expand `PARROT` and `PARROT_AT_DANGER`

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#3640] [General]
Redo the hardcode for parroting to allow defining monster speech as being used on either one or the other so both can be in play. 

Also allow parroting at danger to detect player mutation and only triggers on correct mutations.

This would be represented as new fields within "type": "speech" to change parrot types and valid mutations.

**Comment by Admiral-Daala1541** [comment]
Could you give an example of what this does and how it could be used out of curiosity.

**Reply by ghost** [reply]
A monster can only use parrot, or parrot at danger. So an example would be if it made idle speech that happened outside of combat, and separate lines in combat. A mi-go would be the best way to represent this. It's speech isn't flavored enough, or ferals.

An example of triggering on correct mutation would be if a parrot monster saw you had any mutated anatomy, like animal ears or tentacles or extra arms, and commented on it. I use monsters for human enemies instead of npc's due to problems with npc's so my usage would be detecting profession traits and applying appropriate fear or anger speech that comments on your character.
